packages/blocks/src/api/raw-handling/html-to-blocks.js

```diff
@@ -51,7 +52,8 @@ export function htmlToBlocks( html, handler ) {
 		const { transform, blockName } = rawTransform;
 
 		if ( transform ) {
-			return transform( node, handler );
+			const block = transform( node, handler );
+			return applyFilters( 'blocks.htmlToBlocks', block, node );
```

Comment:
Should we use a private Symbol for now? Or expose a better name for this?

Reply:
I wonder if we should just bundle the class transform?

Reply:
Do it here? I guess we can if you prefer that, but the attribute might not be registered if those other hooks don't run?

Reply:
To be honest, the other hooks should be bundled. I think the main reason they're hooks is to separate the implementation of the block editor code.

Reply:
Ok, done